aws/src/main/java/org/apache/iceberg/aws/glue/IcebergToGlueConverter.java

```diff
@@ -234,8 +259,26 @@ static void setTableInputInformation(
       Optional.ofNullable(properties.get(GLUE_DESCRIPTION_KEY))
           .ifPresent(tableInputBuilder::description);
 
+      List<Column> columns = toColumns(metadata);
+      if (existingTable != null) {
+        List<Column> existingColumns = existingTable.storageDescriptor().columns();
+        Map<String, Column> existingColumnMap =
```

Comment:
What do you think about making this a map of column name to comment here. Since we aren't using any additional information from the existing comments